### ISSUE TITLE
"Flickering" with placeholders using replacements in displaynames

### ISSUE THREAD

**Issue by rymiel** [NEZNAMY/TAB#476]
**Server version**
Velocity 3.1.2

**Plugin version** ("latest" is not a version)
Any build since 5b50c02d93e869864f8eecccb979aec87aab8a54

**Describe the bug**
When using placeholders sent over plugin channels with replacements, the value will rapidly change. Multiple different placeholders using replacements or constantly changing placeholders may be necessary, as it seems to re-send the displayname every time a replacement is processed.

**Expected behavior**
An unchanging placeholder as was the case before 5b50c02d93e869864f8eecccb979aec87aab8a54

**Actual behavior**
Flickering between the default case and the expected case for the given replacement

**Additional info** (errors, screenshots, ...)
Specific commit was hunted down with `git bisect`. More specifically, reverting [these particular changes](https://github.com/NEZNAMY/TAB/commit/5b50c02d93e869864f8eecccb979aec87aab8a54#diff-3e91531442a6bd7ee7403e21ac0f968c77f5672def6a9e69893e491e7cf98308L57-R57) fixes the issue, however I am too unfamiliar with the placeholder implementation to know exactly why

**Comment by NEZNAMY** [comment]
You forgot the most important part - steps to reproduce. I had no issue with replacements.

**Comment by rymiel** [comment]
Reproduced by starting a completely fresh paper and velocity server connected to each other with TAB on velocity and the bukkit-bridge on paper, along with vault, placeholderapi and multiverse (and whatever placeholderapi extensions those needed). Then modified the default TAB config on velocity to add
```yml
placeholder-output-replacements:
  '%multiverse_world_alias%':
      creative: ' ⒯ '
      creative_nether: ' ⒯ '
      the_end: ' ⒮ '
      minigames: ' ⒨ '
      world: ' ⒮ '
      abyss: ' ☣ '
      void: ' ☣ '
      else: '~~~~~~~~~~~'
 ```
 under config.yml, and
 ```yml
 _DEFAULT_:
  tabprefix: "%multiverse_world_alias%%vault-prefix%"
```
under groups.yml.
While standing completely idle, I was able to observe the flicker to the "else" case, which i made wide here to notice easier. I couldn't observe any oddities when the else case was removed. The placeholder itself doesn't return anything different.
(The effect was a lot less glaringly obvious in this minimal reproduction setup, it probably has to do with the amount of placeholders or replacements or network delay). Even if this is specifically because of the multiverse placeholder, it's still a regression I could easily bisect to a single commit, and then to a single change in that commit.
